thirdparty/build-thirdparty.sh

```diff
@@ -1066,13 +1066,15 @@ build_arrow() {
     "${BUILD_SYSTEM}" install
 
     #copy dep libs
-    cp -rf ./jemalloc_ep-prefix/src/jemalloc_ep/dist/lib/libjemalloc_pic.a "${TP_INSTALL_DIR}/lib64/libjemalloc.a"
+    cp -rf ./jemalloc_ep-prefix/src/jemalloc_ep/dist/lib/libjemalloc_pic.a "${TP_INSTALL_DIR}/lib64/libjemalloc_arrow.a"
     cp -rf ./brotli_ep/src/brotli_ep-install/lib/libbrotlienc-static.a "${TP_INSTALL_DIR}/lib64/libbrotlienc.a"
     cp -rf ./brotli_ep/src/brotli_ep-install/lib/libbrotlidec-static.a "${TP_INSTALL_DIR}/lib64/libbrotlidec.a"
     cp -rf ./brotli_ep/src/brotli_ep-install/lib/libbrotlicommon-static.a "${TP_INSTALL_DIR}/lib64/libbrotlicommon.a"
     strip_lib libarrow.a
-    strip_lib libjemalloc.a
+    strip_lib libjemalloc_arrow.a
     strip_lib libparquet.a
+
+    cp -rf "${TP_INSTALL_DIR}/lib64/libjemalloc_arrow.a" "${TP_INSTALL_DIR}/lib64/libjemalloc.a" # TODO delete
```

Comment:
This line will overwrite the static library (libjemalloc.a) built before in the function `build_jemalloc`. Is it right?

Reply:
`build_jemalloc` set the suffix `--with-install-suffix="_doris"`, the compiled static library is `libjemalloc_doris.a`. 

Here `cp -rf libjemalloc_arrow.a libjemalloc.a` is just for compatibility, this line will be deleted later, only `libjemalloc_arrow.a` and `libjemalloc_doris.a` in `thirdparty/install/lib`

![image](https://github.com/apache/doris/assets/13197424/65968632-1e2f-4f56-8529-75291d3ef567)